Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.yml

```diff
@@ -13,6 +13,12 @@ configuration:
   type: 9
   required: true
   hiddenusername: true
+- display: Max. indicators per fetch
+  name: max_indicator_to_fetch
+  type: 0
+  additionalinfo: In case of performance issues (Timeouts), you can limit the number of indicators fetched per fetch.
```

Comment:
```suggestion
  additionalinfo: In case of performance issues leading to timeouts, try limiting the number of indicators retrieved in a fetch run.```